pachyderm/templates/pachd/rbac/worker-role.yaml

```diff
@@ -1,4 +1,4 @@
-{{ if .Values.rbac.create }}
+{{- if .Values.rbac.create -}}
```

Comment:
I _think_ we just want the whitespace chomp at the start, not both

Reply:
Looks like it makes no difference with Helm (maybe Helm trims leading and trailing whitespace?). In the general case, though, one wants it with the Go templating library: https://play.golang.org/p/4XT6hLhX6if

Reply:
So then how come `end` only has one?

Reply:
`end` should only have a single whitespace-chomp, because in order to be a POSIX text file the file must end in a newline (if not empty).

But it is actually at the wrong end of the end: it should be `{{ end -}}`, not `{{- end }}`. Cf. https://play.golang.org/p/U72ztY9pX-h Fixing now …